### ISSUE TITLE
Return IDs list of nested normilized data in result

### ISSUE THREAD

**Issue by AleshaOleg** [paularmstrong/normalizr#218]
I [asked](http://stackoverflow.com/questions/41609432/return-ids-list-of-nested-normilized-data) in StackOverflow about my question, but I don't think so I will get answer there. Now I will 

I want to return IDs of nested data in `result` after normalization.

All data on this issue will be from this [example](https://github.com/paularmstrong/normalizr#quick-start).

After normalizing data I will get this one:

```
{
  result: "123",
  entities: {
    "articles": { 
      "123": { 
        id: "123",
        author: "1",
        title: "My awesome blog post",
        comments: [ "324" ]
      }
    },
    "users": {
      "1": { "id": "1", "name": "Paul" },
      "2": { "id": "2", "name": "Nicole" }
    },
    "comments": {
      "324": { id: "324", "commenter": "2" }
    }
  }
}
```

Is there any way to get IDs of `comments` or `users` during process of normalization? Under this question I mean, is any way that this IDs will go to result object during normalization?

Basically I can do it with `Object.keys(user sW` or `Object.keys(comments)`. But I don't think it's a good idea do it in this way. So, @paularmstrong what you think about it? May be I can help to  implement some method/attribute which can help to make this automatically?

**Comment by paularmstrong** [comment]
Help questions will not be answered via Issues. These are reserved for bug reports and feature needs. Please ask help questions on StackOverflow

**Comment by AleshaOleg** [comment]
@paularmstrong but what if I will implement something to this purpose, and will send pull request? Will you accept it, if you like idea?

**Comment by paularmstrong** [comment]
I don't think so. There's already a very easy way to get the result that you're looking for.
